_includes/payment-order-other-features.md

```diff
@@ -371,6 +377,24 @@ A list of possible operations and their explanation is given below.
             "rel": "failed-paymentorder",
             "contentType": "application/problem+json"
         }
+        {`DEPRECATED DO NOT USE!`
+            "method": "POST",
+            "href": "{{ page.api_url }}/psp/paymentorders/{{ page.payment_order_id }}/captures",
+            "rel": "create-capture",
+            "contentType": "application/json"
+        },
+        {`DEPRECATED DO NOT USE!`
+            "method": "POST",
+            "href": "{{ page.api_url }}/psp/paymentorders/{{ page.payment_order_id }}/cancellations",
```

Comment:
I think the `href` points directly to the payment instrument. Please investigate.

```suggestion
            "href": "{{ page.api_url }}/psp/creditcard/{{ page.payment_id }}/cancellations",
```